RxRealm.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name             = "RxRealm"
-  s.version          = "3.1.0"
+  s.version          = "3.2.0"
```

Comment:
might want this to be a major 4.0,0 bump

Reply:
done